packages/ceramic-http-client/src/ceramic-http-client.ts

```diff
@@ -36,6 +37,24 @@ class CeramicClient {
     return doc
   }
 
+  async _pinDocument (id: string): Promise<Document> {
+    const doc = new Document(id, {}, this._apiUrl)
+    return await doc.pinDocument()
+  }
+
+  async _unpinDocument (id: string): Promise<any> {
+    const doc = new Document(id, {}, this._apiUrl)
+    return await doc.unpinDocument()
+  }
+
+  async _listPinned (docId?: string): Promise<any> {
+    let url = this._apiUrl + '/pin/ls'
+    if (docId !== undefined) {
+      url += docId
+    }
+    return await fetchJson(url)
+  }
+
```

Comment:
Not sure why the methods are different here from what they look like in  `ceramic-core`.

In general `ceramic-core` and `ceramic-http-client` should have the same api (see comments above about the pinning api).